frontend/public/style/_base.scss

```diff
@@ -18,6 +18,13 @@ dd {
 dd:last-child {
   margin-bottom: 0px; }
 
+fieldset {
+  border: 0;
+  margin: 0;
```

Comment:
This should be covered by https://github.com/patternfly/patternfly/blob/7842e225de1884a41160551a890eb6cb3f4541b8/src/patternfly/base/_globals.scss#L31?